### ISSUE TITLE
UnboundLocalError: local variable 'success' referenced before assignment

### ISSUE THREAD

**Issue by oskar-j** [google/oauth2client#307]
Hi, I'm trying to run the example program from https://developers.google.com/youtube/v3/docs/comments/list#examples which downloads comments from youtube. My goal is to download all comments from a particular video on Youtube. I configured the `client_secrets.json` file, but when I run `python dataset2.py --noauth_local_webserver --videoid=pOrW6swvun0` I get an error

```
Traceback (most recent call last):
  File "dataset2.py", line 178, in <module>
    youtube = get_authenticated_service(args)
  File "dataset2.py", line 61, in get_authenticated_service
    credentials = run_flow(flow, storage, args)
  File "C:\Python27\lib\site-packages\oauth2client\util.py", line 142, in positi
onal_wrapper
    return wrapped(*args, **kwargs)
  File "C:\Python27\lib\site-packages\oauth2client\tools.py", line 180, in run_f
low
    if not success:
UnboundLocalError: local variable 'success' referenced before assignment
```

My code is on Gist: https://gist.github.com/oskar-j/861ccdd9253c879aca31 Thanks for any help, this is my first time with the Youtube API.

**Comment by dhermes** [comment]
Thanks for filing! #301 fixed this and #304 will push a new release out to address it.

**Comment by dhermes** [comment]
FYI @nathanielmanistaatgoogle we really need to get #304 out
